documentation/config_file_auth.md

```diff
@@ -135,16 +155,16 @@ If a request matches the rule, Puppet Server checks the request's authenticated
 * **`allow-unauthenticated`**: If this Boolean parameter is set to `true`, Puppet Server allows the request --- even if it can't determine an authenticated name. **This is a potentially insecure configuration** --- be careful when enabling it. A rule with this parameter set to `true` can't also contain the `allow` or `deny` parameters.
 * **`allow`**: This parameter can take a single string value, an array of string values, a single map value with either an `extensions` or `certname` key, or an array of string and map values.
 
-    The string values can be:
+    The string values can contain:
 
     * An exact domain name, such as `www.example.com`.
     * A glob of names containing a `*` in the first segment, such as `*.example.com` or simply `*`.
     * A regular expression surrounded by `/` characters, such as `/example/`.
     * A backreference to a regular expression's capture group in the `path` value, if the rule also contains a `type` value of `regex`. For example, if the path for the rule were `"^/example/([^/]+)$"`, you can make a backreference to the first capture group using a value like `$1.domain.org`.
 
-    The map values can be:
+    The map values can contain:
 
-    * An `extensions` key that specifies a set of matching X.509 extensions. Puppet Server authenticates the request only if each key in the map appears in the request, and each key's value exactly matches.
+    * An `extensions` key that specifies an array of matching X.509 extensions. Puppet Server authenticates the request only if each key in the map appears in the request, and each key's value exactly matches.
```

Comment:
Though there is a great example config above, it might still be nice to show the "multiple possible values for an extension" case either here or above (ie something like `{extensions: {gnarlytimes: [yeah, word, sweet]}}`)